Мелкозёров Леонид

danmerus opened the pull request:
[Посмотреть решение](https://rawgit.com/danmerus/markup-task-5/master/index.html)

danmerus committed:
first version

danmerus committed:
first set of style fixes

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

danmerus committed:
can I dare?

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

danmerus committed:
fall back

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

danmerus committed:
style(2)

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

danmerus committed:
pls

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

danmerus committed:
фиксики

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

danmerus committed:
фиксики

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

danmerus committed:
больше фиксиков

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

danmerus committed:
lf eol

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

danmerus committed:
lf again

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

danmerus committed:
я не понимаю что там с lf

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

danmerus committed:
ops

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

dotokoto commented:
1) Почему чекбоксы уползают наверх, когда в них тыкаешь? D: Очень неудобно
![2017-11-10 18-25-03](https://user-images.githubusercontent.com/8973726/32660406-8960428a-c644-11e7-9b78-5d4f3a6488a6.png)
2) Карточки яростно скачут при наведении. Надо чтобы не скакали, а только открывалось описание
3) в последних двух галереях нет картинок ( Давай больше овощей
4) будет прикольно, если сделаешь выравнивание открытой большой картинки по центру экрана
5) Странные черточки под картинками, которые еще и налезают на текст при его появлении
![2017-11-10 18-31-11](https://user-images.githubusercontent.com/8973726/32660608-63eb7992-c645-11e7-8070-c071ee9568df.png)

dotokoto commented:
6) Переключатели не должны съезжать при открытии карточки
![2017-11-10 18-33-23](https://user-images.githubusercontent.com/8973726/32660712-b3fc92f4-c645-11e7-935a-0d61b51344bc.png)

dotokoto commented:
:tomato:

danmerus committed:
Кажется все

По 4 пункту итак по центру же?
Черточки пофиксил все кроме одной, нет идей откуда она берется...

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

danmerus committed:
space

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

danmerus committed:
end of source newline??

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

danmerus committed:
end of lines

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

danmerus committed:
more lf

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

danmerus committed:
еще 1 попытка

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

dotokoto commented on index.html:
```diff
@@ -3,7 +3,225 @@
+    <input type="radio" name="option" class="radio2">
+    <input type="radio" name="option" class="radio3">
+    <input type="radio" name="option" class="radio4">
+    <div class="block1">
```
не хватает семантики, можно оформить это в `section`

dotokoto commented on index.html:
```diff
@@ -3,7 +3,225 @@
+    <input type="radio" name="option" class="radio4">
+    <div class="block1">
+        <a href="#modal1"><img class="pic1" src="1.jpg" alt="Белокочаннная"></a>
+            <div class="text">Наиболее распространенное холодостойкое влаголюбивое растение семейства
```
А тут можно оформить в `article`

dotokoto commented on index.html:
```diff
@@ -3,7 +3,225 @@
+    <link href="index.css" rel="stylesheet" type="text/css">
 </head>
 <body>
+<div class="spoiler">
```
больше семантики! Тут можно `main` вместо `div`

dotokoto commented on index.css:
```diff
@@ -0,0 +1,153 @@
+    margin: -7% 20px 20px;
+}
+
+.pic3
```
Нужны более семантичные названия классов. Сейчас непонятно, чем они все отличаются и какую роль играют.
И почему их 4? Не хватит двух - для маленьких картинок и для больших на весь экран?

danmerus replied:
почему то разные блоки картинок уезжают по-разному, из-за этого костыли

dotokoto reviewed (changes requested):
Уже лучше! Но еще есть, что поправить

dotokoto commented:
Картинка не влезает в высоту экрана. Надо сделать так, чтобы все картинки влезали независимо от их изначального размера
![2017-11-13 11-02-07](https://user-images.githubusercontent.com/8973726/32712246-60e0dedc-c865-11e7-8d20-15d6b21209cb.png)

danmerus committed:
кое-что еще поправил

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

danmerus committed:
Add files via upload

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

danmerus committed:
spaces

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты
